### PR TITLE
Make all secrets behavior the same way.

### PR DESCRIPTION
**Pull request by ggrocco**
I get an error to have these different behaviors.

### CODE REVIEW

**Comment by rusenask** [comment]
hmm, there's no code in Keel to decode base64 secrets or how would this work?

**Comment by ggrocco** [comment]
> hmm, there's no code in Keel to decode base64 secrets or how would this work?

Secrets need to be encoded anyway in base64, this way is only more easy to manage, because of the local values.yaml doesn't need to have the values encoded. 

I wrote some tools for encoding/decoding files on the console without the need to save encoded https://github.com/ggrocco/snippets.

**Comment by rusenask** [comment]
in this example:

```
  BASIC_AUTH_PASSWORD: {{ .Values.basicauth.password | b64enc }}
```

will keel receive encoded password or plain string?

**Comment by ggrocco** [comment]
> in this example:
> 
> ```
>   BASIC_AUTH_PASSWORD: {{ .Values.basicauth.password | b64enc }}
> ```
> 
> will keel receive encoded password or plain string?

The `.Values.basicauth.password` is a plain string, with `| b64enc` will encode when creating the secrets, and the POD will receive as a plain string!

File: chart/keel/templates/secret.yaml [hunk 1]
```diff
@@ -15,16 +15,16 @@ data:
   AWS_SECRET_ACCESS_KEY:  {{ .Values.ecr.secretAccessKey | b64enc }}
```
Comment (ggrocco) [inline chart/keel/templates/secret.yaml]: @rusenask this was already using this technic

**Comment by rusenask** [comment]
Ah yeah, didn't see that it's actually secret content :) @arthurio does that work with your PR?

**Comment by arthurio** [comment]
@rusenask Yes that works. I just opened: #410 , should we cherry-pick the change here to avoid a double version bump?

**Pull request merged**